util/envDynawo.sh

```diff
@@ -2297,6 +2416,10 @@ case $MODE in
     open_nrt_doc || error_exit "Error during the opening of Dynawo nrt documentation"
     ;;
 
+  run-doc-tests)
```

Comment:
options should be alphabetically ordered

Reply:
Done